test/fuzz/call_graph_test.cpp

```diff
@@ -274,7 +277,9 @@ TEST(CallGraphTest, FunctionInDegree) {
   const auto consumer = nullptr;
 
   const auto context = BuildModule(env, consumer, shader, kFuzzAssembleOption);
-  ASSERT_TRUE(IsValid(env, context.get()));
+  spvtools::ValidatorOptions validator_options;
+  fuzzerutil::IsValidAndWellFormed(context.get(), validator_options,
+                                   kConsoleMessageConsumer);
```

Comment:
Should we ASSERT_TRUE here?

Reply:
This issue occurs a lot in tests.

Reply:
Oh dear - well done for spotting this!  I used regexes to "fix" the tests and obviously messed up.

I believe I have now rectified everything.